packages/angular/src/utils/mfe/mfe-webpack.ts

```diff
@@ -81,6 +81,21 @@ export function shareWorkspaceLibraries(
   };
 }
 
+function collectPackageSecondaries(pkgName: string, packages: string[]) {
+  const pathToPackage = join(workspaceRoot, 'node_modules', pkgName);
+  const secondaries = readdirSync(pathToPackage)
+    .map((file) => join(pathToPackage, file))
+    .filter((file) => lstatSync(file).isDirectory() && file !== 'node_modules')
+    .filter((dir) => existsSync(join(dir, 'package.json')));
+  for (const secondary of secondaries) {
+    const libName = secondary
+      .replace(/\\/g, '/')
+      .replace(/^.*node_modules[/]/, '');
```

Comment:
Might be safer to rely on the `name` specified in the secondary entry point's `package.json`.

Reply:
that's a great point